libs/SalesforceSwiftSDK/readme.md

```diff
@@ -0,0 +1,49 @@
+Mobile SDK 6.1 enhances its Swift offering with PromiseKit adaptations. You can now use promises for these SDK components:
```

Comment:
Probably best not to mention `6.1` here (we already mention this in our `What's New` section. Could you remove `6.1`?